pkg/features/kube_features.go

```diff
@@ -273,6 +280,10 @@ var defaultVersionedFeatureGates = map[featuregate.Feature]featuregate.Versioned
 	TASFailedNodeReplacementFailFast: {
 		{Version: version.MustParse("0.13"), Default: false, PreRelease: featuregate.Alpha},
 	},
+	ManagedJobsNamespaceSelectorAlwaysRespected: {
+		{Version: version.MustParse("0.13"), Default: false, PreRelease: featuregate.Beta},
```

Comment:
```suggestion
		{Version: version.MustParse("0.13"), Default: false, PreRelease: featuregate.Alpha},
```

K8s convention is usually a beta field is default to true.